components/src/core/components/DropdownMenu/DropdownMenu.vue

```diff
@@ -30,18 +30,33 @@ export default defineComponent({
       isActive: false,
     };
   },
+  props: {
```

Comment:
@Super-Chama can you comment on this. I feel it doesn't need a prop but instead it should have the default behavior as this.

Reply:
@thilinasan16 and I had a discussion about this. he agreed to remove it for default behaviour